src/vm/reflectioninvocation.cpp

```diff
@@ -1383,8 +1383,18 @@ FCIMPL4(Object*, RuntimeMethodHandle::InvokeMethod,
         }
 #endif
 
+#ifdef _TARGET_ARM64_
+        ArgLocDesc argLocDescForStructInRegs;
```

Comment:
@jashook I am sorry for the delay, I was OOF for the last four days. Looking at the change again, I am not sure I understand one thing related to my last comment here. It seems that IsHFA now always returns `true`, since you always pass the ArgDestination constructor a non-null `argLocDescForStructInRegs` argument. Is that correct?